src/core/lib/event_engine/ares_resolver.cc

```diff
@@ -702,4 +707,6 @@ void noop_inject_channel_config(ares_channel* /*channel*/) {}
 void (*event_engine_grpc_ares_test_only_inject_config)(ares_channel* channel) =
     noop_inject_channel_config;
 
+bool g_event_engine_grpc_ares_test_only_force_tcp = false;
```

Comment:
will it ever be possible to compile this and the older grpc_ares_wrapper code into the same binary?

If so we would have an ODR violation here?

I suppose we have the same issue with `noop_inject_channel_config` ?

Reply:
ah nevermind I see the name is different